targets/TARGET_ARM_SSG/TARGET_CM3DS_MPS2/rtc_api.c

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2017 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "rtc_api.h"
+#include "device.h"
+#include "cmsis.h"
+
+/**
+ * \defgroup hal_rtc RTC hal functions
+ * @{
+ */
+
+/**
+ * \brief Initialize the RTC peripheral.
+ *        The RTC starts counting from 0x0 at reset and increments
+ *        with a frequency of 1hz (clock source: CLK1HZ).
+ *        The current value can be read through the DATA register
+ */
+void rtc_init(void)
+{
+    CMSDK_RTC->RTCCR |= (1 << CMSDK_RTC_Enable_Pos);
+}
+
+/**
+ * \brief Deinitialize the RTC peripheral
+ *        According to DDI0224B_RTC_PL031_TRM.pdf chapter 3.3.4 there is
+ *        no reason to implement.
+ */
+void rtc_free(void)
+{
+    /* Not supported */
```

Comment:
@bridadan, @0xc0170 The comment above the function is exactly for helping with this question.

 * \brief Deinitialize the RTC peripheral
 *        According to DDI0224B_RTC_PL031_TRM.pdf chapter 3.3.4 there is
 *        no reason to implement.